docs/logs/log-rate.asciidoc

```diff
@@ -0,0 +1,98 @@
+
+[role="screenshot"]
+image::logs/images/analysis-tab-create-ml-job.png[Create machine learning job]
+
+1. Select the time range for the machine learning analysis.
+By default, the analysis uses logs no older than four weeks, and continues indefinitely.
+You cannot change the time range for the analysis after the machine learning job has been created.
+2. Add the indices containing the logs you want to analyze.
+3. Click *Create ML job*.
+
+Even though the machine learning job's time range is fixed,
+you can still use the time filter to adjust the results that are shown in your analysis.
+
+[float]
+[[logs-analysis-view-log-entries]]
+=== View log entries
+
+The *Log rate* page displays the following:
+
+* <<logs-analysis-entries-chart>>
+* <<logs-analysis-anomalies-chart>>
+
+[role="screenshot"]
+image::logs/images/analysis-tab.png[Analysis tab in Logs app in Kibana]
+
+[float]
+[[logs-analysis-entries-chart]]
+==== Log entries chart
+
+The log entries chart shows an overall, color-coded visualization of the log entry rate,
+partitioned according to the value of the Elastic Common Schema (ECS)
+{ecs-ref}/ecs-event.html[`event.dataset`] field.
+
+
+If you have a lot of log partitions, the default view may not be the best way to examine your logs.
+You can filter the data shown by:
+
+* Hovering over a time period to see the log rate for each of the partitions for that period.
+* Clicking on a partition name to show or hide the values for that partition.
+* Hovering over a partition name to highlight just the values for that partition in the chart.
+
+[float]
+[[logs-analysis-anomalies-chart]]
+==== Anomalies charts
+
+The Anomalies chart shows the time period where anomalies were detected in the overall log entry rate across all log partitions.
+The underlying rate values are shown in grey, while the anomalous regions are color-coded and superimposed on top.
+
+When a time period is flagged as anomalous,
+it means that the machine learning algorithms detected something unusual about the log rate in that time period.
+This may be because the log rate was significantly higher than usual, significantly lower than usual,
+or because some other anomalous behavior was detected.
+
+The level of anomaly detected in a time period is color-coded, from red, orange, yellow, to blue.
+Red indicates a critical anomaly level, while blue is a warning level.
+
+To help you further dig into a potential anomaly,
+you can view an anomaly chart for each individual partition:
+
+[role="screenshot"]
+image::logs/images/analysis-tab-partition-anomalies.png[Analysis tab partition anomalies]
+
+Hover over an underlying log rate value to see the average log rate for that time period,
+or hover over an anomalous region to see the partitions that had anomalies in that time period,
+and their anomaly scores.
+Anomaly scores range from 0 (no anomalies) to 100 (critical).
+
+*Analyze in ML* will open the {kibana-ref}/xpack-ml.html[Anomaly Explorer in Machine Learning],
```

Comment:
To change this to active voice, how about:

To analyze the anomalies in more detail, click **Analyze in ML**, which opens the {kibana-ref}/xpack-ml.html[Anomaly Explorer in Machine Learning].